docs/zh/analytics/dashboard/retention.md

```diff
@@ -11,32 +11,32 @@
 ## 数据从哪里来
 留存报告是基于以下 QuickSight 数据集创建的：
 
-- `lifecycle_weekly_view-<app id>-<project id>`，连接到分析引擎（即 Redshift 或 Athena）中的 `clickstream_lifecycle_weekly_view_v1` 视图。
-- `lifecycle_daily_view-<app id>-<project id>`，连接到分析引擎（即 Redshift 或 Athena）中的 `clickstream_lifecycle_daily_view_v1` 视图。
-- 连接到分析引擎中的 `clickstream_retention_view_v1` 视图。
+- `lifecycle_weekly_view-<app id>-<project id>`，连接到分析引擎（即 Redshift 或 Athena）中的 `clickstream_lifecycle_weekly_view_v2` 视图。
+- `lifecycle_daily_view-<app id>-<project id>`，连接到分析引擎（即 Redshift 或 Athena）中的 `clickstream_lifecycle_daily_view_v2` 视图。
+- 连接到分析引擎中的“clickstream_retention_view_v2”视图的“retention_view-<app id>-<project id>”
 
 下面是生成视图的 SQL 命令。
-??? 示例 "SQL 命令"
-    === "Redshift"
-        ```sql title="clickstream_lifecycle_weekly_view_v1.sql"
-        --8<-- "src/analytics/private/sqls/redshift/dashboard/clickstream_lifecycle_weekly_view_v1.sql:6"
-        ```
-        ```sql title="clickstream_lifecycle_daily_view_v1.sql"
-        --8<-- "src/analytics/private/sqls/redshift/dashboard/clickstream_lifecycle_daily_view_v1.sql:6"
-        ```
-        ```sql title="clickstream_retention_view_v1.sql"
-        --8<-- "src/analytics/private/sqls/redshift/dashboard/clickstream_retention_view_v1.sql:6"
-        ```
-    === "Athena"
-        ```sql title="clickstream-lifecycle-weekly-query.sql"
-        --8<-- "src/analytics/private/sqls/athena/clickstream-lifecycle-weekly-query.sql"
-        ```
-        ```sql title="clickstream-lifecycle-daily-query.sql"
-        --8<-- "src/analytics/private/sqls/athena/clickstream-lifecycle-daily-query.sql"
-        ```
-        ```sql title="clickstream-lifecycle-daily-query.sql"
-        --8<-- "src/analytics/private/sqls/athena/clickstream-retention-query.sql"
-        ```
+??? 示例"SQL 命令"
+     === "Redshift"
```

Comment:
Did you have a local running after making this change?

Reply:
You have to revert those format changes. It breaks the code rendering.

<img width="721" alt="image" src="https://github.com/awslabs/clickstream-analytics-on-aws/assets/843303/a8ae8a7f-b396-4c9d-9af2-e2ba1a6ddf75">

Reply:
verified